tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/navigate-agent-dashboard.and.js

```diff
@@ -0,0 +1,11 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible} from '../../utils/driver';
+import {
+    firstAgentList
+  } from '../../pageobjects/agents/agents.page';
+   
+And('The user navigates to the agent dashboard', () => {
+  cy.wait(2000);
```

Comment:
we should get rid of these waits (if we can't do it, maybe we can add a TODO to fix them in the future).

Reply:
we can discuss whats the best aproach